<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jigsaw</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="MyLabel" name="label_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          0</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          1</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          2</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          4</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          5</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          3</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          7</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          8</string>
   </property>
  </widget>
  <widget class="MyLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>          6</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>301</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelsrc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>291</width>
     <height>301</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Original picture&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vediolabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>511</width>
     <height>361</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the vedio area&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Camera loading</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHelp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;get help&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>31</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;select one picture you want&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Select Picture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;array the positions randomly&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Reset Jigsaw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;click to quit the game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelMove">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The less steps you use, the better skill you have&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelnum">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">MyLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
